### PR TITLE
St 129 pending message

### PR DESCRIPTION
**Pull request by duykhoa**
* show popup message in pending service_order page
* redirect to root path after create new service_order

### CODE REVIEW

File: app/assets/stylesheets/service_orders_show_page.css.sass [hunk 1]
```diff
@@ -488,3 +488,29 @@ footer
...
+    width: 500px
+
+    @media (max-width: 500px)
+      width: 100%
+
+    .container
+      padding: 30px 20px
+
+      p
+        line-height: 1.5em
```
Comment (tendousouji) [inline app/assets/stylesheets/service_orders_show_page.css.sass]: add one break line here please :star2:

Comment (tendousouji) [inline app/assets/stylesheets/service_orders_show_page.css.sass]: add new line here please :star2: thank you :star2:

File: app/views/service_orders/show.html.slim [hunk 2]
```diff
@@ -16,3 +16,14 @@ section.social
...
+- if @service_order.pending?
+  .section#pending-service-order
+    .popup-pending
+      .container.text-left
+        p Hi, #{ @service_order.your_name }!
+        p Thank you for using TenLuaWeb
+        p We are processing your Love Story.
+        p Keep calm and come back in 24 hours.
+        p Best regards,
+        p Khoa Tran
```
Comment (tendousouji) [inline app/views/service_orders/show.html.slim]: new line here too :coffee: thank you

Comment (duykhoa) [inline app/views/service_orders/show.html.slim]: no need, em :100:

File: app/assets/stylesheets/service_orders_show_page.css.sass (hunk 1 above)
Comment (duykhoa) [inline app/assets/stylesheets/service_orders_show_page.css.sass]: no need, em :100:

**Pull request merged**